DigitalLearningSolutions.Web.Tests/ViewModels/TrackingSystem/Delegates/DelegateGroups/GroupDelegatesViewModelTests.cs

```diff
@@ -0,0 +1,75 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.ViewModels.TrackingSystem.Delegates.DelegateGroups
+{
+    using System.Linq;
+    using DigitalLearningSolutions.Data.Models.DelegateGroups;
+    using DigitalLearningSolutions.Data.Tests.TestHelpers;
+    using DigitalLearningSolutions.Web.Models.Enums;
+    using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.DelegateGroups;
+    using FluentAssertions;
+    using FluentAssertions.Execution;
+    using NUnit.Framework;
+
+    public class GroupDelegatesViewModelTests
+    {
+        private readonly DelegateGroupsSideNavViewModel expectedNavViewModel =
+            new DelegateGroupsSideNavViewModel("Group name", DelegateGroupPage.Delegates);
+
+        private readonly GroupDelegate[] groupDelegates =
+        {
+            GroupTestHelper.GetDefaultGroupDelegate(firstName: "a", lastName: "Surname"),
+            GroupTestHelper.GetDefaultGroupDelegate(firstName: "b", lastName: "Surname"),
+            GroupTestHelper.GetDefaultGroupDelegate(firstName: "c", lastName: "Surname"),
+            GroupTestHelper.GetDefaultGroupDelegate(firstName: "d", lastName: "Surname"),
+            GroupTestHelper.GetDefaultGroupDelegate(firstName: "e", lastName: "Surname"),
+            GroupTestHelper.GetDefaultGroupDelegate(firstName: "f", lastName: "Surname"),
+            GroupTestHelper.GetDefaultGroupDelegate(firstName: "g", lastName: "Surname"),
+            GroupTestHelper.GetDefaultGroupDelegate(firstName: "h", lastName: "Surname"),
+            GroupTestHelper.GetDefaultGroupDelegate(firstName: "i", lastName: "Surname"),
+            GroupTestHelper.GetDefaultGroupDelegate(firstName: "j", lastName: "Surname"),
+            GroupTestHelper.GetDefaultGroupDelegate(firstName: "k", lastName: "Surname"),
+            GroupTestHelper.GetDefaultGroupDelegate(firstName: "l", lastName: "Surname"),
+            GroupTestHelper.GetDefaultGroupDelegate(firstName: "m", lastName: "Surname"),
+            GroupTestHelper.GetDefaultGroupDelegate(firstName: "n", lastName: "Surname"),
+            GroupTestHelper.GetDefaultGroupDelegate(firstName: "o", lastName: "Surname")

```

Comment:
These tests aren't touching the database are they?
It would be a bit tidier to use NBuilder, and would allow you to easily set the properties your checking in this file rather than in a separate one (e.g. GroupId, Name).